src/pages/SignUp/SignUp.js

```diff
@@ -1,8 +1,136 @@
-import React from "react";
-import "./SignUp.scss";
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
```

Comment:
동일한 패키지에서 import하기 때문에 함께 import하시면 됩니다.
```suggestion
import { Link, useNavigate } from 'react-router-dom';
```

Reply:
수정 완료!